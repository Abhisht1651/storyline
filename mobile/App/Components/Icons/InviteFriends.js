import React from 'react'
import { View, Text } from 'react-native'

// Styles
import styles from '../Styles/IconStyles/InviteFriendsStyles'
import InviteFriendsSVG from '../../Images/Icons/icon-invite-friends.svg'

const InviteFriends = () => {
  return (
    <View style={styles.cardBox}>
      <View style={styles.profileIconBox}>
        <InviteFriendsSVG />
      </View>
      <View style={styles.TextBox}>
        <Text style={styles.TextStyles}>Invite Friends</Text>
      </View>
    </View>
  )
}

export default InviteFriends
