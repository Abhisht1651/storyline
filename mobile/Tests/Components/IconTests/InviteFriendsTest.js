import React from 'react'
import 'react-native'
import renderer from 'react-test-renderer'

import InviteFriends from '../../../App/Components/Icons/InviteFriends'

test('InviteFriends Icon component renders correctly', () => {
  const tree = renderer.create(<InviteFriends />).toJSON()
  expect(tree).toMatchSnapshot()
})
