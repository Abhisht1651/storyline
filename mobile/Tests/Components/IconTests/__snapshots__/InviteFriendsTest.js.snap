// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InviteFriends Icon component renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "transparent",
      "flexDirection": "row",
      "height": 26,
      "opacity": 1,
      "width": 150,
    }
  }
>
  <View
    style={
      Object {
        "height": 26,
        "width": 23,
      }
    }
  >
    <SvgMock />
  </View>
  <View
    style={
      Object {
        "height": 20,
        "marginLeft": 15,
        "marginTop": 5,
        "width": 114,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#1F1F60",
          "fontFamily": "BananaGrotesk-Medium",
          "fontSize": 18,
          "letterSpacing": -0.12,
          "opacity": 1,
          "textAlign": "left",
        }
      }
    >
      Invite Friends
    </Text>
  </View>
</View>
`;
